### PR TITLE
+add camera and cylinder, New "Add Non-Mesh" and "Add Mesh" toolbar tabs.

### PR DESCRIPTION
**Pull request by Giv0**

### CODE REVIEW

File: src/editor/scene/scene-factory.ts [hunk 1]
```diff
@@ -94,6 +94,13 @@ export default class SceneFactory {
 
         return SceneManager.EnvironmentHelper;
     }
+	
+	public static CreateCamera (editor: Editor): Camera {
+        const camera =  new BABYLON.FreeCamera('New Camera', new BABYLON.Vector3(0, 0, 0), editor.core.scene, true);
+        this.AddToGraph(editor, camera);
```
Comment (julien-moreau) [inline src/editor/scene/scene-factory.ts:105]: Please also add tags to camera like in other functions:
Tags.AddTagsTo(camera, 'added');

File: src/editor/scene/scene-factory.ts [hunk 2]
```diff
@@ -94,6 +94,13 @@ export default class SceneFactory {
 
         return SceneManager.EnvironmentHelper;
     }
+	
+	public static CreateCamera (editor: Editor): Camera {
```
Comment (julien-moreau) [inline src/editor/scene/scene-factory.ts]: Please add comments

File: src/editor/scene/scene-factory.ts [hunk 3]
```diff
@@ -244,6 +251,13 @@ export default class SceneFactory {
 
         return mesh;
     }
+	
+	public static CreateCylinder (editor: Editor): Mesh {
```
Comment (julien-moreau) [inline src/editor/scene/scene-factory.ts:264]: Please add comments

File: src/editor/scene/scene-factory.ts [hunk 4]
```diff
@@ -244,6 +251,13 @@ export default class SceneFactory {
...
+        const mesh = Mesh.CreateCylinder('New Cylinder', 1, 1, 1, 16, editor.core.scene);
+        this.AddToGraph(editor, mesh);
```
Comment (julien-moreau) [inline src/editor/scene/scene-factory.ts:266]: Same, please add tags:
Tags.AddTagsTo(mesh, 'added');

File: src/editor/scene/scene-factory.ts (hunk 1 above)
Comment (givowo) [reply src/editor/scene/scene-factory.ts:105]: I don't understand what you mean by this

File: src/editor/scene/scene-factory.ts (hunk 2 above)
Comment (givowo) [reply src/editor/scene/scene-factory.ts]: Done!

File: src/editor/scene/scene-factory.ts (hunk 3 above)
Comment (givowo) [reply src/editor/scene/scene-factory.ts:264]: Done!

File: src/editor/scene/scene-factory.ts (hunk 4 above)
Comment (givowo) [reply src/editor/scene/scene-factory.ts:266]: I don't understand what you mean by this

Comment (julien-moreau) [reply src/editor/scene/scene-factory.ts:266]: You are right it is not needed, sorry :)

**Comment by julien-moreau** [comment]
All good!
Please just remove the file "BabylonJS Editor Setup 3.1.3.exe" from your commits, these builds should not be included in the repository has they are too heavy
Thanks! :)

**Comment by Giv0** [comment]
I deleted the setup file and updated my repo to be up-to-date with your sound commits.

**Comment by julien-moreau** [comment]
Hey thanks but you can't do that, I'll have conflicts when rebasing the branch feature/graph from master. Please remove these commits as they are coming in master soon.
Also, can you please drop your commit instead of removing the file and commit ?
If you don't drop the commit, the .exe will still be in the git tree and users that will clone the reprository will download the exe :(

**Comment by Giv0** [comment]
Oh, ok.  I'm pretty sure I added it on it's own commit, so that shouldn't
be an issue

**Comment by julien-moreau** [comment]
@Giv0 any news?
I can still se the commits and still can't merge :(

**Comment by Giv0** [comment]
I reverted my commits to delete and create the setup file.  Sorry for taking a while, I have limited access to github. (and an actual pc in general) ;)

**Comment by Giv0** [comment]
Oh, good it looks like it was merged. I had updated it a little while back
sorry for not saying anything.

**Comment by julien-moreau** [comment]
No worry and thanks for your contribution :)

**Comment by Giv0** [comment]
:)

**Pull request merged**